nifi-toolkit/nifi-toolkit-encrypt-config/src/main/groovy/org/apache/nifi/properties/ConfigEncryptionTool.groovy

```diff
@@ -1043,12 +1050,12 @@ class ConfigEncryptionTool {
      * @param plainProperties the NiFiProperties instance containing the raw values
      * @return the NiFiProperties containing protected values
      */
-    private NiFiProperties encryptSensitiveProperties(NiFiProperties plainProperties) {
+    private NiFiProperties encryptSensitiveProperties(NiFiProperties plainProperties, String keyHex) {
```

Comment:
Rename `keyHex` to `keyHexOrKeyId` to indicate this can support AWS KMS as well.

Reply:
Renamed.